pkgs/apple/container/registry.yaml

```diff
@@ -6,7 +6,7 @@ packages:
     description: A tool for creating and running Linux containers using lightweight virtual machines on a Mac. It is written in Swift, and optimized for Apple silicon
     version_constraint: "false"
     version_overrides:
-      - version_constraint: semver("<= 0.6.0")
+      - version_constraint: semver("< 0.7.0")
```

Comment:
I don't think this change has meaning.

https://github.com/apple/container/releases

Versions:

- 0.7.1
- 0.7.0
- 0.6.0

The evaluation result of `<= 0.6.0` and `< 0.7.0` are same in this case.